src/main/resources/assets/skyblocker/lang/en_us.json

```diff
@@ -123,104 +123,27 @@
-  "text.autoconfig.skyblocker.option.quickNav.button10.uiTitle" : "UI Title",
-  "text.autoconfig.skyblocker.option.quickNav.button10.clickEvent" : "Click event",
-  "text.autoconfig.skyblocker.option.quickNav.button11" : "Button 11",
-  "text.autoconfig.skyblocker.option.quickNav.button11.render" : "Render",
-  "text.autoconfig.skyblocker.option.quickNav.button11.item" : "Item",
-  "text.autoconfig.skyblocker.option.quickNav.button11.item.itemName" : "Item name",
-  "text.autoconfig.skyblocker.option.quickNav.button11.item.count" : "Item Count",
-  "text.autoconfig.skyblocker.option.quickNav.button11.item.nbt" : "NBT",
-  "text.autoconfig.skyblocker.option.quickNav.button11.uiTitle" : "UI Title",
-  "text.autoconfig.skyblocker.option.quickNav.button11.clickEvent" : "Click event",
-  "text.autoconfig.skyblocker.option.quickNav.button12" : "Button 12",
-  "text.autoconfig.skyblocker.option.quickNav.button12.render" : "Render",
-  "text.autoconfig.skyblocker.option.quickNav.button12.item" : "Item",
-  "text.autoconfig.skyblocker.option.quickNav.button12.item.itemName" : "Item name",
-  "text.autoconfig.skyblocker.option.quickNav.button12.item.count" : "Item Count",
-  "text.autoconfig.skyblocker.option.quickNav.button12.item.nbt" : "NBT",
-  "text.autoconfig.skyblocker.option.quickNav.button12.uiTitle" : "UI Title",
-  "text.autoconfig.skyblocker.option.quickNav.button12.clickEvent" : "Click event",
+  "text.autoconfig.skyblocker.category.quickNav": "Quick Navigation",
+  "text.autoconfig.skyblocker.option.quickNav.enableQuickNav": "Enable Quick Navigation",
+  "text.autoconfig.skyblocker.option.quickNav.button1": "Button 1",
+  "text.autoconfig.skyblocker.option.quickNav.button1.render": "Render",
+  "text.autoconfig.skyblocker.option.quickNav.button1.item": "Item",
+  "text.autoconfig.skyblocker.option.quickNav.button1.item.itemName": "Item name",
+  "text.autoconfig.skyblocker.option.quickNav.button1.item.count": "Item Count",
+  "text.autoconfig.skyblocker.option.quickNav.button1.item.nbt": "NBT",
+  "text.autoconfig.skyblocker.option.quickNav.button1.uiTitle": "UI Title",
+  "text.autoconfig.skyblocker.option.quickNav.button1.clickEvent": "Click event",
+  "text.autoconfig.skyblocker.option.quickNav.button2": "Button 2",
+  "text.autoconfig.skyblocker.option.quickNav.button3": "Button 3",
+  "text.autoconfig.skyblocker.option.quickNav.button4": "Button 4",
+  "text.autoconfig.skyblocker.option.quickNav.button5": "Button 5",
+  "text.autoconfig.skyblocker.option.quickNav.button6": "Button 6",
+  "text.autoconfig.skyblocker.option.quickNav.button7": "Button 7",
+  "text.autoconfig.skyblocker.option.quickNav.button8": "Button 8",
+  "text.autoconfig.skyblocker.option.quickNav.button9": "Button 9",
+  "text.autoconfig.skyblocker.option.quickNav.button10": "Button 10",
+  "text.autoconfig.skyblocker.option.quickNav.button11": "Button 11",
+  "text.autoconfig.skyblocker.option.quickNav.button12": "Button 12",
```

Comment:
also instead of this only use 
`"text.autoconfig.skyblocker.option.quickNav.button": "Button %d"`
and for the text.translate part you can use the args like this
`Text.translatable("text.autoconfig.skyblocker.option.quickNav.button", 11)`